soh/soh/Enhancements/mods.cpp

```diff
@@ -1441,5 +1442,6 @@ void InitMods() {
     RegisterPatchHandHandler();
     RegisterHurtContainerModeHandler();
     RegisterPauseMenuHooks();
+    RegisterShufflePots();
```

Comment:
Probably better to move the registering to `RandomizerRegisterHooks` in `hook_handlers.cpp`.